practice/practice-02/src/styles/layouts/_welcome.scss

```diff
@@ -240,13 +251,15 @@
     }
 
     &__lines-background {
-      @include setAbsoluteAndLocation($top: -150px, $right: 0px);
+      @include set-absolute-and-location($top: -150px, $right: 0px);
+
       @include size(313px, 718px);
```

Comment:
Two include blocks should not have empty line

Reply:
Fixed.